src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -278,16 +290,19 @@ public class ReleaseService : IReleaseService
             return amendment;
         }
 
-        private async Task<Either<ActionResult, Release>> CopyReleaseRolePermissions(Guid originalReleaseId, Release amendment)
+        private async Task<Either<ActionResult, Release>> CopyReleaseRoles(Guid originalReleaseId, Release amendment)
         {
+            // Copy all current roles apart from Prerelease Users to the Release amendment.
             var newRoles = _context
                 .UserReleaseRoles
-                .AsQueryable()
-                .IgnoreQueryFilters() // For auditing purposes, we also want to migrate release roles that have Deleted set
-                .Where(releaseRole =>
-                    !releaseRole.SoftDeleted
-                    && releaseRole.ReleaseId == originalReleaseId)
-                .Select(releaseRole => releaseRole.CopyForAmendment(amendment));
+                // For auditing purposes, we also want to migrate release roles that have Deleted set (when a role is
+                // manually removed from a Release as opposed to SoftDeleted, which is only set when a Release is
+                // deleted)
+                .IgnoreQueryFilters()
+                .Where(releaseRole => releaseRole.ReleaseId == originalReleaseId 
+                                      && releaseRole.Role != ReleaseRole.PrereleaseViewer)
```

Comment:
I'm not 100%, but I think we might want to keep the `SoftDeleted` check here. Possible you've already explored this, in which case leave it out - but thought I'd raise it.

I think there may have been a distinction between `SoftDeleted` and `Deleted`. I think `Deleted` implies a user has removed the permission, and we keep an audit trail, but `SoftDeleted` implies an overarching object like the release has been removed. I think that's why the original code filtered them out.